libs/application/templates/european-health-insurance-card/src/lib/helpers/applicantHelper.ts

```diff
@@ -56,6 +56,17 @@ export function getFullName(
   return null
 }
 
+export function getPlasticExpiryDate(resp: CardResponse): Date | undefined {
+  try {
```

Comment:
Why do we need try/catch here?

Reply:
if the Date conversion would fail.. i.e. expiryDate is some gibberish